### PR TITLE
Update README with Genshin module information (#51)

### LINKED ISSUE #51: [Enhancement]: Update README with Genshin module information
### My idea

Add information from https://github.com/Gwardia-Czapli/GwardiaHub/discussions/24#discussioncomment-9145332 to `README.md` (to `for genshin impact players` section).

### PR DESCRIPTION
**Pull request by Miszaal**
check my commit guys

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,12 @@ We want to create website with Django, that will be useful to as many as three d
   - place to submit feedback about meetings,
   - assignments given on meetings.
 - For Genshin Impact players:
-  - TBD
+  - Few ideas to enhance the game:
```
Comment (Norbiros) [inline README.md]: Remove this line

File: README.md [hunk 2]
```diff
@@ -19,7 +19,12 @@ We want to create website with Django, that will be useful to as many as three d
...
+  - wish simulator(and list of the characters you've collected already)
```
Comment (Norbiros) [inline README.md:22]: Missing space

**Review by Norbiros — CHANGES_REQUESTED** [review]
Elaborate more UwU (especially for people that never played Genshin).

File: README.md (hunk 1 above)
Comment (Miszaal) [reply README.md]: why?

**Review by goteusz-maszyk — APPROVED** [review]

**Review by Sanielia — APPROVED** [review]
apruf

**Review by hexxcatee — APPROVED** [review]
👍👍

**Review by Norbiros — CHANGES_REQUESTED** [review]
I already told @Miszaal what he needs to change.

**Pull request closed without merging**